src/test/java/com/sugarcrm/candybean/automation/webdriver/WebDriverSystemTest.java

```diff
@@ -102,24 +98,23 @@ public void screenshotTest() throws Exception {
 	@Ignore
 	@Test
 	public void startStopRestartTest() throws Exception {
```

Comment:
This test either is testing too many things or isn't very clear to me the incentive behind it.
1) Why are we saving the URL from the `iface` every time?
2) Why are we reseting `iface` to `FirefoxInterface`, calling `getUrl()`, and restarting with this interface? Why is `iface` reset to `ChromeInterface` afterward? This part is particularly confusing to me
3) Probably need a comment in the `finally` block to indicate that we're starting the interface, so that the test doesn't crash from trying to stop it in the `tearDown()` method.

If the above questions cannot be answered and since this test is ignored, we need a ticket to fix up this mess.